tools/topology/sof/pipe-volume-demux-playback.m4

```diff
@@ -32,7 +53,7 @@ C_CONTROLBYTES(DEMUX, PIPELINE_ID,
 	, , ,
 	CONTROLBYTES_MAX(, 304),
 	,
-	DEMUX_priv)
+	demux_priv)
```

Comment:
where was DEMUX_priv defined? Should it be removed? Or does this change below in another patch?

Reply:
yes it should be removed and I'll make a follow up patch for it